DependencyInjection/SensioFrameworkExtraExtension.php

```diff
@@ -43,6 +43,12 @@ public function load(array $configs, ContainerBuilder $container)
         if ($config['router']['annotations']) {
             $annotationsToLoad[] = 'routing.xml';
 
+            if (class_exists('Symfony\Component\ExpressionLanguage\ExpressionLanguage') && class_exists('Symfony\Component\Security\Core\Authorization\ExpressionLanguage')) {
+                $container->setAlias('sensio_framework_extra.converter.doctrine.orm.expression_language', new Alias('sensio_framework_extra.converter.doctrine.orm.expression_language.default', false));
+            } else {
```

Comment:
While this is correct, you can't test this case. Maybe this is more of a proposition for the DI component though, but it would be really neat if this could be mocked (thus probably wrap the class exists checks)